contracts/hardhat.config.ts

```diff
@@ -1,5 +1,4 @@
 import { HardhatUserConfig } from "hardhat/config";
-import "hardhat-deploy";
 import "@nomicfoundation/hardhat-toolbox";
```

Comment:
Just a yagni thing or specific complaint?

Reply:
I have to look and see what that module does specifically. 
I usually use ethers from hardhat for the deployment process and that is not needed (at least as far as I know for how I usually set it up) however, I just need to research the module and understand what it does.

If it provides benefits to the deployment process I am all for it. Additionally, when I first cloned the repo; it was throwing errors at me left and right since I had to setup some editor config and tweak some NPM modules to make the nesting be understandable and not throw.